test/fixtures/dialects/mysql/analyze_table.sql

```diff
@@ -0,0 +1,45 @@
+ANALYZE TABLE some_table;
+ANALYZE TABLE some_table1, some_table2;
+
+ANALYZE NO_WRITE_TO_BINLOG TABLE some_table;
+ANALYZE NO_WRITE_TO_BINLOG TABLE some_table1, some_table2;
+
+ANALYZE LOCAL TABLE some_table;
+ANALYZE LOCAL TABLE some_table1, some_table2;
+
+ANALYZE TABLE some_table
+UPDATE HISTOGRAM ON some_col;
+ANALYZE TABLE some_table
+UPDATE HISTOGRAM ON some_col1, some_col2;
```

Comment:
I find these statements confusing to grok in a quick glance.

they are two statements over four lines.

I would suggest one if the following:
- collapsing to two lines
- separating with a newline.

Reply:
FYI this applies to all test cases in this PR btw, not just the ones highlighted above.

Reply:
fixed this now 👍